ScreenCapturerExample/README.md

```diff
@@ -0,0 +1,11 @@
+# Twilio Video ScreenCapturer Example
+
+> NOTE: `TVIScreenCapturer` is deprecated on iOS 12.0 and above due to performance issues on wide-color devices. If you wish to share the contents of the screen we recommend that you use [ReplayKit](https://developer.apple.com/documentation/replaykit) instead. We are currently working on a ReplayKit example [app](https://github.com/twilio/video-quickstart-swift/pull/287).
```

Comment:
The intention is that the PR which adds ReplayKitExample will update the link here.